### PR TITLE
Add blog post: LibrePCB 1.0 Roadmap

### PR DESCRIPTION
**Pull request by ubruhin**
Rendered: https://librepcb.org/_branches/blogpost-roadmap-1_0/blog/2023-05-11_roadmap_1.0/

@dbrgn A review would be appreciated, but just let me know if you're short on time ;)

### CODE REVIEW

File: content/blog/2023-05-15_roadmap_1.0/index.adoc [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+As mentioned in the
+link:{{< relref "blog/2023-04-13_ngi0_grant/index.adoc" >}}[previous blog post],
+I'll give you a short update about the status of the upcoming release which
+is currently in development.
+
+Time for 1.0!
+-------------
+
+Actually the plan was that the next release after LibrePCB 0.1.7 will have
```
Comment (dbrgn) [inline content/blog/2023-05-15_roadmap_1.0/index.adoc:15]: I think "The original plan" would sound better than "Actually the plan".

File: content/blog/2023-05-15_roadmap_1.0/index.adoc [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+the version number 0.2. But thanks to the
+link:{{< relref "blog/2023-04-13_ngi0_grant/index.adoc" >}}[NGI0 grant] I'm
+now able to spend much more time for LibrePCB, allowing to include more
+features in the next release than initially planned.
+
+While the current lack of important features like slotted pads still represent
+a show-stopper for LibrePCB in various projects, I'm convinced that the
+features now planned for the next release will make LibrePCB usable for
+much more PCB projects out there. I think this big step should be
+represented by the version number so it will be bumped to 1.0.
```
Comment (dbrgn) [inline content/blog/2023-05-15_roadmap_1.0/index.adoc:25]: This makes it sound a bit as if "current lack" applies to 1.0 as well. However, slotted pads are already implemented in 1.0, if I'm not mistaken. And thermal reliefs were another big showstopper.

Maybe you could change the wording like this?

> While the lack of important features in version 0.1 – like slotted pads or thermal reliefs – still represent
a show-stopper for LibrePCB in various projects, I'm convinced that the
features now planned for the next release will make LibrePCB usable for
much more PCB projects out there.

File: content/blog/2023-05-15_roadmap_1.0/index.adoc [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  (https://github.com/LibrePCB/LibrePCB/pull/1149[#1149])
+- IPC D-356A netlist export
+  (https://github.com/LibrePCB/LibrePCB/pull/1081[#1081])
+- Fully configurable colors
+  (https://github.com/LibrePCB/LibrePCB/pull/1086[#1086])
+
+Highlight: 3D Viewer & STEP Export (https://github.com/LibrePCB/LibrePCB/pull/1156[#1156])
+------------------------------------------------------------------------------------------
+
+At least visually the coolest feature is the 3D board viewer!
```
Comment (dbrgn) [inline content/blog/2023-05-15_roadmap_1.0/index.adoc:79]: Either add a comma:

> At least visually, the coolest feature is the 3D board viewer!

Or maybe:

> From the visual point of view, the coolest new feature is the 3D board viewer!

**Review by dbrgn — COMMENTED** [review]
Sorry, I missed the original notification. I still left a few comments, in case you want to improve the blogpost.

**Comment by ubruhin** [comment]
Thanks @dbrgn! I committed the suggestions to the master branch :+1:

**Pull request merged**